devenv/utils/mine_btc.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+num_blocks=$1
+dir="$(dirname "$0")"
+
+if [[ -z "$num_blocks" ]]; then
+    echo "Usage: ./script.sh [num_blocks]"
+    exit 1
+fi
+
+btc_address=$(source $dir/get_credentials.sh | jq -r '.credentials["0"].bitcoin.p2wpkh.address')
```

Comment:
We need also bitcoins for p2pkh:

btc_address=$(source $dir/get_credentials.sh | jq -r '.credentials["0"].bitcoin.p2pkh.address')

Reply:
Added this to https://github.com/stacks-network/sbtc/pull/175